### ISSUE TITLE
BLV loss

### ISSUE THREAD

**Issue by danzhang0615** [grantword8/BLV#4]
Hi,

i have a question regarding the reason of putting abs() for the variation added to the logit. The rare class logits will receive more positive additive values, this is actually opposite to the practice of logit adjustment ([69] cited by your paper as well).

also, the eq. (3) in the paper is different to the code implementation. 

looking forward to your reply and thanks.

**Comment by grantword8** [comment]
Sorry for the late reply, 

According to my understanding, the paper logit adjustment [69] aims to mitigate the adverse effects of overconfidence in head categories, and it is not contradictory to our approach.

You are correct, we made some minor mistakes while organizing and simplifying the code, and we have now made the necessary corrections. 

Thank you very much for your suggestions.

**Comment by userid623** [comment]
> Hi,
> 
> i have a question regarding the reason of putting abs() for the variation added to the logit. The rare class logits will receive more positive additive values, this is actually opposite to the practice of logit adjustment ([69] cited by your paper as well).
> 
> also, the eq. (3) in the paper is different to the code implementation.
> 
> looking forward to your reply and thanks.

In fact, this paper is very similar to a CVPR '22 paper GCL. I am even inclined to consider it a case of plagiarism.
Maybe the Section3.2 in GCL can answer your question.

GCL: [Long-tailed visual recognition via gaussian clouded logit adjustment](https://openaccess.thecvf.com/content/CVPR2022/papers/Li_Long-Tailed_Visual_Recognition_via_Gaussian_Clouded_Logit_Adjustment_CVPR_2022_paper.pdf)
